api/tests/routes/native/v1/openapi_test.py

```diff
@@ -164,7 +164,6 @@ def test_public_api(client):
                 "BookingOfferExtraData": {
                     "properties": {
                         "ean": {"nullable": True, "title": "Ean", "type": "string"},
-                        "isbn": {"nullable": True, "title": "Isbn", "type": "string"},
                     },
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏Top ça :+1: 
Il faut penser à faire la PR sur le repo native pour mettre à jour `gen.ts`